tests/test_sns/test_subscriptions_boto3.py

```diff
@@ -425,61 +425,179 @@ def test_subscribe_invalid_filter_policy():
                 "FilterPolicy": json.dumps({"store": [str(i) for i in range(151)]})
             },
         )
-    except ClientError as err:
+
         err.response["Error"]["Code"].should.equal("InvalidParameter")
```

Comment:
These lines should be un-indented one level. They are currently inside the `with pytest.raises`-block, so when the previous call fails, these lines are never executed.

Reply:
Thanks, it's fixed now.